src_types/common/anyString$wSelector.d.ts

```diff
@@ -0,0 +1,4 @@
+declare namespace $w {
```

Comment:
I know that what we said in the spec for this one, I just want it to go on the record that I think it's a very bad idea to allow users to invoke $w on public/* files, and we should at least mark it as deprecated or as a warning

Reply:
i also think we should have some warning about it